drivers/disk/sdmmc_stm32.c

```diff
@@ -881,6 +897,11 @@ static struct stm32_sdmmc_priv stm32_sdmmc_priv_1 = {
 	.hsd = {
 		.Instance = (MMC_TypeDef *)DT_INST_REG_ADDR(0),
 		.Init.ClockEdge = SDMMC_CLOCK_EDGE_RISING,
+#ifdef SDIO_CLOCK_BYPASS_DISABLE
+		.Init.ClockBypass = DT_INST_PROP(0, clk_bypass)
+						? SDIO_CLOCK_BYPASS_ENABLE
+						: SDIO_CLOCK_BYPASS_DISABLE,
+#endif
```

Comment:
```suggestion
#elif defined(SDMMC_CLOCK_BYPASS_DISABLE)
```
would be more readable

Reply:
done, thanks